### DISCUSSION TITLE
Urgent: preview does not work in production when I need to edit images

### DISCUSSION THREAD

**Discussion by Evraldouvci** [filamentphp/filament#2457] [Help]
Locally I have no worries but when I go into production the images no longer load so that I can modify them!
![Capture d’écran 2022-05-18 160146](https://user-images.githubusercontent.com/92441009/169089098-8ccc18a7-539b-4918-8073-7c8fe36f85ca.jpg)

**Comment by danharrin** [comment]
You sent no console errors, no code, how are we supposed to help you solve this?

Btw, marking issues as **urgent** doesn't get you free help any quicker 😉

**Comment by Evraldouvci** [comment]
config filesystems 
'disks' => [

        'local' => [
            'driver' => 'local',
            'root' => storage_path('app'),
        ],

        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
        ],

        's3' => [
            'driver' => 's3',
            'key' => env('AWS_ACCESS_KEY_ID'),
            'secret' => env('AWS_SECRET_ACCESS_KEY'),
            'region' => env('AWS_DEFAULT_REGION'),
            'bucket' => env('AWS_BUCKET'),
            'url' => env('AWS_URL'),
            'endpoint' => env('AWS_ENDPOINT'),
        ],

    ],


my domain root is public_html

**Reply by danharrin** [reply]
What console errors have you recieved?

**Reply by Evraldouvci** [reply]
no error message the preview just doesn't load in the edit view of the filament admin panel as shown in the screenshot

**Reply by danharrin** [reply]
Okay, this is definitely a config problem on your end that I can't really help debug

**Reply by rajatpatelz** [reply]
@Evraldouvci check value of `APP_URL` in `.env` file, this should match with your domain name / IP Address from which you are accessing the application.
